parquet-tools/src/main/java/parquet/tools/command/SizeCommand.java

```diff
@@ -0,0 +1,133 @@
+/**
+ * Copyright 2013 ARRIS, Inc.
```

Comment:
@swapnilushinde, don't worry about it, this is good stuff. There are always things to fix, which is why everything gets reviewed.

There's also no way you would know about the headers. Copying them from the other commands is reasonable, we are just moving to the Apache one so new files should use it. Thanks for offering to update the others, but we are already talking with ARRIS to get that done so we'll handle it separately.